controllers/supervisor/supervisor.py

```diff
@@ -461,6 +461,8 @@ def get_robot_touch_ball(self):
                 for id in range(constants.NUMBER_OF_ROBOTS):
                     if message[2 * id + team] == 1:
                         rc[team][id] = True
+                    if rc[team][id]:
+                        self.sound_speaker(0,1)
```

Comment:
```suggestion
                        self.sound_speaker(KICK_SOUND, 1)
```

Reply:
I probably forgot to change this one, hehe. Thanks!